R/lump.R

```diff
@@ -93,6 +93,44 @@ fct_lump <- function(f, n, prop, w = NULL, other_level = "Other",
   }
 }
 
+#' Title
```

Comment:
Could be a little more descriptive 😛 — but I think it would be better to `@rdname fct_lump` so all the lump functions are documented together.